### PR TITLE
Remove redundant methods from ExprTest

### PR DESCRIPTION
**Pull request by mbasmanova**
Remove methods that are available in the base class VectorTestBase.

### CODE REVIEW

**Review by xiaoxmeng — APPROVED** [review]
Thanks for the change!

File: velox/expression/tests/ExprTest.cpp [hunk 1]
```diff
@@ -575,50 +575,9 @@ class ExprTest : public testing::Test, public VectorTestBase {
...
-    return vectorMaker_->flatVector<T>(size, valueAt, isNullAt);
-  }
-
-  FlatVectorPtr<StringView> makeFlatVector(
-      const std::vector<std::string>& data) {
-    return vectorMaker_->flatVector(data);
-  }
-
   template <typename T>
   FlatVectorPtr<T> makeAllNullFlatVector(vector_size_t size) {
```
Comment (kagamiori) [inline velox/expression/tests/ExprTest.cpp:579]: Could this function be removed too?

File: velox/expression/tests/ExprTest.cpp [hunk 2]
```diff
@@ -1395,7 +1314,8 @@ TEST_F(ExprTest, cseEncodings) {
 
   auto indices = makeIndices({0, 0, 0, 1, 1});
   auto b = wrapInDictionary(indices, 5, makeFlatVector<int32_t>({11, 15}));
-  auto c = wrapInDictionary(indices, 5, makeFlatVector({"apple", "banana"}));
+  auto c = wrapInDictionary(
+      indices, 5, makeFlatVector<std::string>({"apple", "banana"}));
```
Comment (kagamiori) [inline velox/expression/tests/ExprTest.cpp:1318]: Why are we changing this? Can the type be inferred automatically?

**Review by kagamiori — APPROVED** [review]
LGTM. I left a few questions.

File: velox/expression/tests/ExprTest.cpp (hunk 1 above)
Comment (mbasmanova) [reply velox/expression/tests/ExprTest.cpp:579]: makeAllNullFlatVector is not available in VectorTestBase, but I can move it.

File: velox/expression/tests/ExprTest.cpp (hunk 2 above)
Comment (mbasmanova) [reply velox/expression/tests/ExprTest.cpp:1318]: It didn't compile without explicit template parameter.

File: velox/expression/tests/ExprTest.cpp (hunk 1 above)
Comment (kagamiori) [reply velox/expression/tests/ExprTest.cpp]: I mean I saw makeAllNullFlatVector simply calls `vectorMaker_.allNullFlatVector<T>(size)`. So I thought we can just use vectorMaker_.allNullFlatVector() instead?

File: velox/vector/tests/VectorTestBase.cpp [hunk 3]
```diff
@@ -58,6 +57,19 @@ BufferPtr VectorTestBase::makeIndices(
...
 }
 
+BufferPtr VectorTestBase::makeIndices(
+    const std::vector<vector_size_t>& indices) const {
+  auto size = indices.size();
+  BufferPtr indicesBuffer =
+      AlignedBuffer::allocate<vector_size_t>(size, pool());
+  auto rawIndices = indicesBuffer->asMutable<vector_size_t>();
+
+  for (int i = 0; i < size; i++) {
```
Comment (kgpai) [inline velox/vector/tests/VectorTestBase.cpp:67]: I think you can use memcpy or std::copy  something like : 
`memcpy (rawIndices, indices.data(), indicies.size () * sizeof(vector_size_t)); `

**Review by kgpai — APPROVED** [review]

**Pull request closed without merging**